captcha-validation-service/src/main/java/io/mosip/esignet/captcha/controller/CaptchaController.java

```diff
@@ -0,0 +1,41 @@
+package io.mosip.esignet.captcha.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.Errors;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import io.mosip.esignet.captcha.dto.CaptchaRequestDTO;
+import io.mosip.esignet.captcha.dto.MainRequestDTO;
+import io.mosip.esignet.captcha.exception.CaptchaException;
+import io.mosip.esignet.captcha.exception.InvalidRequestCaptchaException;
+import io.mosip.esignet.captcha.exception.InvalidRequestParameterException;
+import io.mosip.esignet.captcha.spi.CaptchaService;
+import io.mosip.esignet.captcha.util.DataValidationUtil;
+import lombok.extern.slf4j.Slf4j;
+
+@RestController
+@Slf4j
+public class CaptchaController {
+
+	private static final String VALIDATE = "validate";
+
+
+	@Autowired
+	private CaptchaService captchaService;
+
+	@PostMapping(path = "/validatecaptcha", consumes = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<?> validateCaptcha(@Validated @RequestBody MainRequestDTO<CaptchaRequestDTO> captchaRequest,
+			Errors errors) throws InvalidRequestParameterException, CaptchaException, InvalidRequestCaptchaException {
+		log.info("sessionId", "idType", "id",
```

Comment:
first 3 argument is unnecessary here , directly log the message as below:
log.info("In pre-registration captcha controller to validate the recaptcha token");

Reply:
done the changes as specified